src/ev-matrix.c

```diff
@@ -29,6 +29,7 @@ static CmAccount *account;
 static GCancellable *cancel;
 static GListModel *joined_rooms;
 static GPtrArray *pushers;
+static GRegex *room_regex;
```

Comment:
Need to cleanup in `ev_matrix_destroy`

Reply:
thanks! I did *briefly* wonder where the other stuff got cleaned up